src/mobu/github_config.py

```diff
@@ -0,0 +1,130 @@
+"""Config for GitHub application integrations."""
+
+from enum import StrEnum
+from textwrap import dedent
+
+from pydantic import Field, field_validator
+from pydantic_settings import BaseSettings
+
+from .models.user import User
+
+
+class GafaelfawrScope(StrEnum):
+    """Available Gafaelfawr scopes."""
+
+    exec_notebook = "exec:notebook"
+    exec_portal = "exec:portal"
+    read_image = "read:image"
+    read_tap = "read:tap"
```

Comment:
I would just not validate this. Scopes can be dynamically defined in the Gafaelfawr configuration, so there is no standard list of scopes (they may differ by installation or environment), only a set of conventions that we use for the RSP. We also add new ones not that infrequently. The drawback is that if there's a typo, mobu will fail at startup when trying to create the Gafaelfawr user, but I think that's usually a fairly obvious error.